russian/instagram/demon-story/demon.md

```diff
@@ -0,0 +1,5 @@
+- Хе-хе. Очнулся таки, путешественничек. Давай там долго не засиживайся, а то скоро вернется Ведьмочка-Танечка и ты можешь
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.